erpnext/stock/doctype/stock_entry/stock_entry.json

```diff
@@ -455,7 +455,7 @@
    "read_only": 1
   },
   {
-   "depends_on": "eval:doc.purpose === \"Send to Subcontractor\"",
+   "depends_on": "eval:((doc.purpose === \"Send to Subcontractor\") || (doc.purpose === \"Material Transfer\" && doc.is_return))",
```

Comment:
Two other fields missed here: `subcontracting_order` and `purchase_order`

There is repetition of the condition. Can we use a function to evaluate this?

Reply:
I don't think so function can be used for `eval`.